package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@adobe/plugin-hooks",
-  "version": "0.1.1",
+  "version": "0.2.0-beta1",
```

Comment:
@AjazSumaiya are we publishing this as beta?

Reply:
to test i was thinking of publishing beta first

Reply:
To publish as a legitimate beta, we'll need to add the beta tag in the [.github/workflows
/publish-to-npm.yml](https://github.com/adobe/plugin-hooks/blob/develop/.github/workflows/publish-to-npm.yml#L22-L24) file (and later revert it).

Reply:
updated the tag